src/planner/binder/statement/bind_drop.cpp

```diff
@@ -61,6 +61,9 @@ BoundStatement Binder::Bind(DropStatement &stmt) {
 		}
 		auto entry = config.storage_extensions.begin();
 		auto &storage_extension = entry->second;
+		if (storage_extension->drop_database_extension_function == nullptr) {
+			throw BinderException("DROP DATABASE is not supported in this extension.");
```

Comment:
You have the extension name in `entry->first` so you can use it in the error message

Reply:
Done